prog/test/vm_group.rb

```diff
@@ -3,14 +3,15 @@
 require "net/ssh"
 
 class Prog::Test::VmGroup < Prog::Test::Base
-  def self.assemble(storage_encrypted: true, test_reboot: true)
+  def self.assemble(storage_encrypted: true, test_reboot: true, boot_images: Option::BootImages.map { _1.name })
```

Comment:
If the boot images don't change while the app is running, it would be good to store this `Option::BootImages.map { _1.name }` as a frozen constant, to avoid allocating every time this is called.